### ISSUE TITLE
tabBar items size

### ISSUE THREAD

**Issue by kirapwn** [AppGyver/steroids#864]
Hello,
When using tabBar without any background (icon), it should be possible to remove the element that is pushing tab Title down. Or would `display: none` work but it is just not documented?

Would be great if we had a template for the tab-bar-item placed in common/ folder to customize.

**Comment by harsaharsa** [comment]
Did you try out `display: none`? Sounds like a legit request in any case. Related: https://github.com/AppGyver/steroids/issues/726

**Comment by kirapwn** [comment]
I did try display: none, !important and other CSS. Line height, height, width, position etc. Nothing seems to have effect. While debugging you can't even see this part as HTML element (within <html> tags). I assume it is emulating this html to work native on device. But that is putting strict limitations to customizing.

Right now I switched to custom html tabBar.

**Comment by harsaharsa** [comment]
Yup, it's interpreting the CSS to style the native element using a framework called Pixate Freestyle. There's plenty of limitations compared to the full control you get with HTML – we're evaluating options on how to proceed.
